Src/Model/Adapter/Database.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Oat\Model\Adapter;
+
+use PDO;
+
+class Database implements AdapterInterface
+{
+
+    private $pdo;
+    private $configPath = __DIR__ . '/../../../config/database.ini';
+
+    /**
+     * Database constructor.
+     */
+    public function __construct()
+    {
+        $config = parse_ini_file($this->configPath);
+        $this->pdo = new PDO('mysql:dbname=' . $config['dbname'] . ';host=' . $config['host'], $config['username'], $config['password']);
+        $this->pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+    }
+
+    /**
+     * get one user with the ID
+     * @param $id
+     * @return array
+     * @throws \Exception
+     */
+    public function showOne($id)
+    {
+        $id = $id + 1;
+        $request = $this->pdo->prepare('SELECT * FROM users WHERE id = ?');
```

Comment:
`*` fetch all columns, you may specify field you want to avoid strange behaviour in the future

Reply:
done